### DISCUSSION TITLE
Feature request: separate inline suggestion flags for comments vs code

### DISCUSSION THREAD

**Discussion by tangentstorm** [community/community#55543] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

I find that when I'm writing code, copilot sometimes seems to be reading my mind and suggests exactly what I want -- sometimes even before I figure it out myself.

BUT when I'm writing comments, the accuracy rate is much, much lower. After all, usually if I'm writing a comment, it's because the code isn't easy to understand on its own.

So, I often find myself pausing to think in the middle of writing a comment, and then a completely un-helpful and distracting suggestion pops up.

It would be nice if I could only get comment suggestions when I press the suggestion hotkey, but continue to get code completion suggestions automatically... So: please consider making separate flags for that. Thanks!

**Comment by d2x** [comment]
Related to [Possibility to disable comment suggestion](https://github.com/orgs/community/discussions/8062)
